src/pytorch_lightning/__init__.py

```diff
@@ -36,3 +36,34 @@ def _detail(self: Any, message: str, *args: Any, **kwargs: Any) -> None:
 
 # for compatibility with namespace packages
 __import__("pkg_resources").declare_namespace(__name__)
+
+LIGHTNING_LOGO: str = """
+                    ####
+                ###########
+             ####################
+         ############################
+    #####################################
+##############################################
+#########################  ###################
+#######################    ###################
+####################      ####################
+##################       #####################
+################        ######################
+#####################        #################
+######################     ###################
+#####################    #####################
+####################   #######################
+###################  #########################
+##############################################
+    #####################################
+         ############################
+             ####################
+                  ##########
+                     ####
+"""
+
+
+def cli_lightning_logo() -> None:
```

Comment:
nit: the function name sounds redundant. Could be updated to:

```suggestion
def print_logo() -> None:
```

Do we need this addition though?

Reply:
lets do it follow-up PR